src/gui/StelDialog.cpp

```diff
@@ -351,7 +354,15 @@ void StelDialog::askColor()
 	{
 		vColor = Vec3f(c.redF(), c.greenF(), c.blueF());
 		StelApp::getInstance().getStelPropertyManager()->setStelPropertyValue(propName, QVariant::fromValue(vColor));
-		StelApp::getInstance().getSettings()->setValue(iniName, StelUtils::vec3fToStr(vColor));
+		if (moduleName.length()==0)

```

Comment:
Why not `moduleName.isEmpty()`?

Reply:
Yes, another way to ask the same question.